specs-go/version.go

```diff
@@ -81,16 +97,16 @@ func (v version) String() string {
 }
 
 // IsGreaterThan checks with a version is greater than the specified version.
-func (v version) IsGreaterThan(o version) bool {
+func (v version) isGreaterThan(o version) bool {
```

Comment:
Since the type is private we should not have to do this. If we do, please also update the docstring.

Reply:
I'd prefer to explicitly set them public to avoid possible confusions. Updated the docstrings.

Reply:
Should be good to merge as soon as you push this.

Reply:
pushed. Please, merge, thanks!